dpbench/benchmarks/dbscan/dbscan_numba_npr.py

```diff
@@ -79,7 +76,7 @@ def get_neighborhood(n, dim, data, eps, ind_lst, sz_lst, assignments):
 def compute_clusters(n, min_pts, assignments, sizes, indices_list):
     nclusters = 0
     nnoise = 0
-    for i in nb.prange(n):
+    for i in range(n):
```

Comment:
This loop cannot be executed in parallel. Hence changing from prange to range. this is fine